src/sentry/static/sentry/less/base.less

```diff
@@ -251,48 +282,22 @@ abbr[title] {
   text-decoration: underline dotted; // 2
 }
 
-//
-// Address style set to `bolder` in Firefox 4+, Safari, and Chrome.
-//
-
 b,
 strong {
   font-weight: bold;
 }
 
-//
-// Address styling not present in Safari and Chrome.
-//
-
 dfn {
   font-style: italic;
 }
 
-//
-// Address variable `h1` font-size and margin within `section` and `article`
-// contexts in Firefox 4+, Safari, and Chrome.
-//
-
-h1 {
-  font-size: 2em;
-  margin: 0.67em 0;
-}
-
-//
-// Address styling not present in IE 8/9.
-//
-
 mark {
   background: #ff0;
   color: #000;
 }
 
-//
-// Address inconsistent and variable font size in all browsers.
-//
-
 small {
-  font-size: 80%;
+  font-size: 14px;
 }
```

Comment:
This change looks odd but I took these styles from another Bootstrap file that was overriding this stuff.